### ISSUE TITLE
Server Error 500 (Internal Server Error)

### ISSUE THREAD

**Issue by mowald62** [snipe/snipe-it#1528]
Hello, I installed snipe IT successfully however when i try to launch the app, I get the Server Error 500 (Internal Server Error). 
Below are the logs
[2015-12-27 12:27:24] production.ERROR: exception 'ErrorException' with message 'mcrypt_encrypt(): Key of size 36 not supported by this algorithm. Only keys of sizes 16, 24 or 32 supported' in C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Encryption\Encrypter.php:78
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleError(2, 'mcrypt_encrypt(...', 'C:\\xampp\\htdocs...', 78, Array)
#1 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Encryption\Encrypter.php(78): mcrypt_encrypt('rijndael-256', 'Change_this_key...', 's:40:"b40d04c22...', 'cbc', '\xF9\xAC\x92\x15\xE0\x96\xF8\xA2\xD4`\\\xDB+\xB7\x0E...')
#2 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Encryption\Encrypter.php(57): Illuminate\Encryption\Encrypter->padAndMcrypt('b40d04c2299b201...', '\xF9\xAC\x92\x15\xE0\x96\xF8\xA2\xD4`\\\xDB+\xB7\x0E...')
#3 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Cookie\Guard.php(118): Illuminate\Encryption\Encrypter->encrypt('b40d04c2299b201...')
#4 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Cookie\Guard.php(51): Illuminate\Cookie\Guard->encrypt(Object(Illuminate\Http\RedirectResponse))
#5 C:\xampp\htdocs\snipe\vendor\stack\builder\src\Stack\StackedHttpKernel.php(23): Illuminate\Cookie\Guard->handle(Object(Illuminate\Http\Request), 1, true)
#6 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(641): Stack\StackedHttpKernel->handle(Object(Illuminate\Http\Request))
#7 C:\xampp\htdocs\snipe\public\index.php(49): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2015-12-27 12:39:27] production.ERROR: exception 'ErrorException' with message 'mcrypt_encrypt(): Key of size 36 not supported by this algorithm. Only keys of sizes 16, 24 or 32 supported' in C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Encryption\Encrypter.php:78
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleError(2, 'mcrypt_encrypt(...', 'C:\\xampp\\htdocs...', 78, Array)
#1 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Encryption\Encrypter.php(78): mcrypt_encrypt('rijndael-256', 'Change_this_key...', 's:40:"3f4adf239...', 'cbc', '\x95\x9Cc\xD7\xB5]\x06x|u\xA4\x9B\xD9\x83\x84...')
#2 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Encryption\Encrypter.php(57): Illuminate\Encryption\Encrypter->padAndMcrypt('3f4adf239c7de3c...', '\x95\x9Cc\xD7\xB5]\x06x|u\xA4\x9B\xD9\x83\x84...')
#3 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Cookie\Guard.php(118): Illuminate\Encryption\Encrypter->encrypt('3f4adf239c7de3c...')
#4 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Cookie\Guard.php(51): Illuminate\Cookie\Guard->encrypt(Object(Illuminate\Http\RedirectResponse))
#5 C:\xampp\htdocs\snipe\vendor\stack\builder\src\Stack\StackedHttpKernel.php(23): Illuminate\Cookie\Guard->handle(Object(Illuminate\Http\Request), 1, true)
#6 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(641): Stack\StackedHttpKernel->handle(Object(Illuminate\Http\Request))
#7 C:\xampp\htdocs\snipe\public\index.php(49): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2015-12-27 12:40:16] production.ERROR: exception 'ErrorException' with message 'mcrypt_encrypt(): Key of size 36 not supported by this algorithm. Only keys of sizes 16, 24 or 32 supported' in C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Encryption\Encrypter.php:78
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleError(2, 'mcrypt_encrypt(...', 'C:\\xampp\\htdocs...', 78, Array)
#1 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Encryption\Encrypter.php(78): mcrypt_encrypt('rijndael-256', 'Change_this_key...', 's:40:"645c8581c...', 'cbc', '\xE2\x9Bb\xC12n\xB5\x01\x84yeg\xA8b\xB4...')
#2 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Encryption\Encrypter.php(57): Illuminate\Encryption\Encrypter->padAndMcrypt('645c8581cc02d63...', '\xE2\x9Bb\xC12n\xB5\x01\x84yeg\xA8b\xB4...')
#3 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Cookie\Guard.php(118): Illuminate\Encryption\Encrypter->encrypt('645c8581cc02d63...')
#4 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Cookie\Guard.php(51): Illuminate\Cookie\Guard->encrypt(Object(Illuminate\Http\RedirectResponse))
#5 C:\xampp\htdocs\snipe\vendor\stack\builder\src\Stack\StackedHttpKernel.php(23): Illuminate\Cookie\Guard->handle(Object(Illuminate\Http\Request), 1, true)
#6 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(641): Stack\StackedHttpKernel->handle(Object(Illuminate\Http\Request))
#7 C:\xampp\htdocs\snipe\public\index.php(49): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2015-12-27 12:45:09] production.ERROR: exception 'ErrorException' with message 'mcrypt_encrypt(): Key of size 36 not supported by this algorithm. Only keys of sizes 16, 24 or 32 supported' in C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Encryption\Encrypter.php:78
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleError(2, 'mcrypt_encrypt(...', 'C:\\xampp\\htdocs...', 78, Array)
#1 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Encryption\Encrypter.php(78): mcrypt_encrypt('rijndael-256', 'Change_this_key...', 's:40:"2bc300d17...', 'cbc', 'yA%\xF2\xCA\xEF\xAA\x05\xB6\xF3i\xCD\xD3\x96\xB3...')
#2 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Encryption\Encrypter.php(57): Illuminate\Encryption\Encrypter->padAndMcrypt('2bc300d178cc665...', 'yA%\xF2\xCA\xEF\xAA\x05\xB6\xF3i\xCD\xD3\x96\xB3...')
#3 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Cookie\Guard.php(118): Illuminate\Encryption\Encrypter->encrypt('2bc300d178cc665...')
#4 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Cookie\Guard.php(51): Illuminate\Cookie\Guard->encrypt(Object(Illuminate\Http\RedirectResponse))
#5 C:\xampp\htdocs\snipe\vendor\stack\builder\src\Stack\StackedHttpKernel.php(23): Illuminate\Cookie\Guard->handle(Object(Illuminate\Http\Request), 1, true)
#6 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(641): Stack\StackedHttpKernel->handle(Object(Illuminate\Http\Request))
#7 C:\xampp\htdocs\snipe\public\index.php(49): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2015-12-27 12:56:38] production.ERROR: exception 'ErrorException' with message 'mcrypt_encrypt(): Key of size 36 not supported by this algorithm. Only keys of sizes 16, 24 or 32 supported' in C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Encryption\Encrypter.php:78
Stack trace:
#0 [internal function]: Illuminate\Exception\Handler->handleError(2, 'mcrypt_encrypt(...', 'C:\\xampp\\htdocs...', 78, Array)
#1 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Encryption\Encrypter.php(78): mcrypt_encrypt('rijndael-256', 'Change_this_key...', 's:40:"8a0a449af...', 'cbc', '\t\xFB\x9624\x1F\xDC\x84XQ\xE6\x03j\x8E\x96...')
#2 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Encryption\Encrypter.php(57): Illuminate\Encryption\Encrypter->padAndMcrypt('8a0a449af162639...', '\t\xFB\x9624\x1F\xDC\x84XQ\xE6\x03j\x8E\x96...')
#3 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Cookie\Guard.php(118): Illuminate\Encryption\Encrypter->encrypt('8a0a449af162639...')
#4 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Cookie\Guard.php(51): Illuminate\Cookie\Guard->encrypt(Object(Illuminate\Http\RedirectResponse))
#5 C:\xampp\htdocs\snipe\vendor\stack\builder\src\Stack\StackedHttpKernel.php(23): Illuminate\Cookie\Guard->handle(Object(Illuminate\Http\Request), 1, true)
#6 C:\xampp\htdocs\snipe\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(641): Stack\StackedHttpKernel->handle(Object(Illuminate\Http\Request))
#7 C:\xampp\htdocs\snipe\public\index.php(49): Illuminate\Foundation\Application->run()
#8 {main} [] []

**Comment by jaskegreen** [comment]
Likewise for me. I just did a second install on my Mac Pro server. I followed the install process exactly as stated and now getting a 500 error when directed to the login page.

I'm running on OSX 10.8 with Web services on Server.app, Apache 2, and PHP 5.6. Here are my most recent logs:
[log-apache2handler-2016-01-13.txt](https://github.com/snipe/snipe-it/files/90628/log-apache2handler-2016-01-13.txt)
